source/contracts/trading/Trade.sol

```diff
@@ -14,7 +14,7 @@ import 'libraries/CashAutoConverter.sol';
 
 
 contract Trade is CashAutoConverter, ReentrancyGuard {
-    uint256 private constant MINIMUM_GAS_NEEDED = 300000;
+    uint256 private constant MINIMUM_GAS_NEEDED = 500000;
```

Comment:
I'm assuming this is up-to-date with latest?  Is it possible to author a test that will fail if gas exceeds this amount?

Reply:
Yea this is closer to the latest tests but still an estimate. Will add a test for this